src/components/Form/Label.js

```diff
@@ -1,165 +1,145 @@
-    padding: '7px 0 5px',
-    ...convertStyleToRem(sansSerifRegular22),
-    lineHeight: pxToRem(27),
-    minHeight: pxToRem(fieldHeight),
-    color: colors.text,
-    width: '100%',
-    appearance: 'none',
-    outline: 'none',
-    borderRadius: 0,
-    backgroundColor: '#fff',
-    border: 'none',
-    textAlign: 'left',
-    cursor: 'pointer',
-    borderBottom: `solid ${colors.disabled} ${borderWidth}px`,
-    ':focus': {
-      borderColor: colors.primary
-    }
-  }),
   select: css({
     position: 'absolute',
-    top: pxToRem(labelHeight),
+    top: pxToRem(LABEL_HEIGHT),
     left: 0,
     right: 0,
     bottom: 0,
-    opacity: 0,
-    ':focus + svg': {
-      fill: colors.primary
-    }
+    opacity: 0
   }),
   selectArrow: css({
     position: 'absolute',
     right: 0,
     top: pxToRem(28),
-    pointerEvents: 'none',
-    fill: colors.disabled
-  }),
-  selectArrowBlack: css({
-    fill: '#000'
-  }),
-  selectArrowWhite: css({
-    fill: '#fff'
-  }),
-
-  white: css({
-    backgroundColor: 'transparent',
-    color: '#fff',
-    borderColor: '#fff',
-    ':focus': {
-      borderColor: '#fff'
-    }
-  }),
-  black: css({
-    backgroundColor: 'transparent',
-    color: '#000',
-    borderColor: '#000',
-    ':focus': {
-      borderColor: '#000'
-    }
+    pointerEvents: 'none'
   })
 }
 
-export const Label = ({ top, focus, error, text, black, white, children }) => {
-  const labelTextStyle = merge(
-    styles.labelText,
-    top && styles.labelTextTop,
-    focus && styles.labelTextFocused,
-    error && styles.labelTextError,
-    white && styles.white,
-    black && styles.black
-  )
+export const Label = ({
+  top,
+  focus,
+  error,
+  text,
+  children,
+  Element = 'span',
+  field,
+  ...props
+}) => {
+  const [colorScheme] = useColorContext()
+  const labelTextStyle = merge(styles.labelText, top && styles.labelTextTop)
+  const isSelect = Element === 'select'
+  const styleRules = useMemo(() => {
+    return {
+      field: css({
+        borderColor: colorScheme.getCSSColor('disabled'),
+        ':focus': {
+          borderColor: colorScheme.getCSSColor('primary')
+        }
+      }),
+      select: css({
+        ':focus + svg': {
+          fill: colorScheme.getCSSColor('primary')
+        }
+      })
+    }
+  }, [colorScheme])
+
+  if (field) {
+    return (
+      <>
+        <Element
+          {...props}
+          {...merge(
+            styles.field,
+            styleRules.field,
+            isSelect && styles.select,
+            isSelect && styleRules.select
+          )}
+          {...(error
+            ? colorScheme.set('color', 'error')
+            : focus
+            ? colorScheme.set('color', 'primary')
+            : colorScheme.set('color', 'text'))}
+        >
+          {children}
+        </Element>
+        {Element === 'select' ? (
+          <svg
+            key='arrow'
+            {...styles.selectArrow}
+            {...colorScheme.set('fill', 'text')}
```

Comment:
```suggestion
            {...colorScheme.set('fill', 'disabled')}
```

Like the virtual:
<img width="1034" alt="Screenshot 2020-10-27 at 17 38 29" src="https://user-images.githubusercontent.com/410211/97333179-8a950c80-187b-11eb-8f02-4c2d609012ec.png">

Reply:
good catch!